examples/ensemble/plot_random_forest_regression_multioutput.py

```diff
@@ -0,0 +1,71 @@
+"""
+============================================================
+Comparing random forests and the multi-output meta estimator
+============================================================
+
+An example to compare multi-output regression with random forest and
+the :ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator.
+
+This example illustrates the use of the
+:ref:`multioutput.MultiOutputRegressor <_multiclass>` meta-estimator
+to perform multi-output regression. A random forest regressor is used,
+which supports multi-output regression natively, so the results can be
+compared.
```

Comment:
I still don't completely get the intuition but your graph suggests that this is a matter of underfitting that data using lesser max_depth. If you increase the `max_depth` the points close to the center go away.

Also, I find it strange that the meta-estimator performs better than the native RandomForest regressor, Is this because of the different score functions or something else?